packages/ui/src/ui/pages/query-tracker/module/api.ts

```diff
@@ -406,13 +409,41 @@ export function getQueryResultMeta(
     return (_dispatch, getState) => {
         const state = getState();
         const {stage} = getQueryTrackerRequestOptions(state);
-        return ytApiV4Id.getQueryResults(YTApiId.readQueryResults, {
+        return ytApiV4Id.getQueryResults(YTApiId.getQueryResults, {
             parameters: {stage, query_id, result_index},
             setup: getQTApiSetup(),
         });
     };
 }
 
+export function getQueryResultMetaList(
+    query_id: string,
+    inds: number[],
+): ThunkAction<Promise<SplitedBatchResults<QueryResultMeta>>, RootState, any, any> {
+    return async (_dispatch, getState) => {
```

Comment:
If `_dispatch` is not use it might be just a function returning a promise (not ThunkAction)